databricks/koalas/frame.py

```diff
@@ -373,7 +373,7 @@ def _reduce_for_stat_function(self, sfun, name, axis=None, numeric_only=False):
         elif axis in ('columns', 1):
             # Here we execute with the first 1000 to get the return type.
             # If the records were less than 1000, it uses pandas API directly for a shortcut.
-            limit = 1000
+            limit = get_option("compute.shortcut_limit")
```

Comment:
this test too https://github.com/databricks/koalas/commit/d354de649ebdea0f75705e2923a1bfb355371ca5#diff-d3f67c2eab2a8dd3df7283426995819fR60